spec/spec_helper.rb

```diff
@@ -1,3 +1,4 @@
+require 'pundit/matchers'
```

Comment:
Please make sure that Coveralls stays on top. That is a requirement.

Reply:
Will do 👍